src/GeneratedTestComponent/docs/index.story.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import { storySettings } from './storySettings';
+import LiveCodeExample from '../../../stories/utils/Components/LiveCodeExample';
 
-import GeneratedTestComponent from '../../src/GeneratedTestComponent';
+import GeneratedTestComponent from '..';
 
 export default {
   category: storySettings.category,
   storyName: storySettings.storyName,
 
   component: GeneratedTestComponent,
-  componentPath: '../../src/GeneratedTestComponent/GeneratedTestComponent.js',
+  componentPath: '../GeneratedTestComponent.js',
```

Comment:
```suggestion
  componentPath: '..',
```
this should be enough

Reply:
fixed